alf/algorithms/entropy_target_algorithm.py

```diff
@@ -32,7 +33,18 @@
 
 
 @alf.configurable
-class EntropyTargetAlgorithm(Algorithm):
+def EntropyTargetAlgorithm(action_spec,
+                           initial_alpha=0.1,
+                           skip_free_stage=False,
+                           max_entropy=None,
+                           target_entropy=None,
+                           very_slow_update_rate=0.001,
+                           slow_update_rate=0.01,
+                           fast_update_rate=np.log(2),
+                           min_alpha=1e-4,
+                           average_window=2,
+                           debug_summaries=False,
+                           name="EntropyTargetAlgorithm"):
```

Comment:
Should add an explanation that _NestedEntropyTargetAlgorithm will be created for nested action_spec.